NodeJS/index.js

```diff
@@ -0,0 +1,33 @@
+const { initialize } = require('unleash-client');
+require('dotenv').config()
+
+const apiUrl = process.env.UNLEASH_API_URL
+const apiToken = process.env.UNLEASH_API_TOKEN
+const flag = 'example-flag'
+
+// Initialize the Unleash client
+const unleash = initialize({
+  url: apiUrl,
+  appName: 'codesandbox-node',
+  metricsInterval: 1000,
+  customHeaders: {
+    Authorization: apiToken,
+  },
+});
+
+unleash.on('ready', () => {
```

Comment:
do we wait for the ready event in other SDKs? I thought we're following the most basic example of initialize + check status inside interval

Reply:
... Not that I know. But it seems like the idiomatic way to do it (we also show the event in the docs).